### PR TITLE
Fix tests due to repoze.sphinx.autointerface

### PR DESCRIPTION
**Pull request by icemac**
See https://github.com/repoze/repoze.sphinx.autointerface/issues/16
and https://github.com/zopefoundation/meta/pull/96

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -37,19 +38,26 @@ commands =
 
 [testenv:docs]
 basepython = python3
+skip_install = false
+# Until repoze.sphinx.autointerface supports Sphinx 4.x we cannot use it:
+deps = Sphinx < 4
 extras =
     docs
+commands_pre =
```
Comment (mgedmin) [inline tox.ini:46]: What's this for?

**Review by mgedmin — APPROVED** [review]
LGTM

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini:46]: It ensures that a possibly inherited `command_pre` (from `[testenv]`) does not get executed. (`command_pre` is used in the `zope-product` template but the snipped for the documentation is shared between all templates.)

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**